umbral/openssl.py

```diff
@@ -63,13 +66,12 @@ def _bn_is_on_curve(check_bn, curve_nid: int):
     return check_sign == 1 and range_check == -1
 
 
-def _int_to_bn(py_int: int, curve_nid: int=None):
+def _int_to_bn(py_int: int, curve_nid: int=None, is_consttime=True):
     """
     Converts the given Python int to an OpenSSL BIGNUM. If a curve_nid is
     provided, it will check if the Python integer is within the order of that
     curve. If it's not within the order, it will raise a ValueError.
-
-    TODO: Add flag for consttime
+    is_consttime set to True by default to use constant time ops.
```

Comment:
This part of the docstring doesn't quite make sense.  Probably better is:

`:param is_consttime (or better name): Whether to tell the backend to use constant-time operations for this instance.